components/Starknet/modules/architecture-and-concepts/pages/network-architecture/transaction-life-cycle.adoc

```diff
@@ -54,6 +54,7 @@ A `REJECTED` transaction is stored in the mempool. You cannot send another trans
 |`ACCEPTED_ON_L2` |The transaction was executed and entered an actual created block on L2.
 |`ACCEPTED_ON_L1` |The transaction was accepted on Ethereum.
 .2+|*Execution* |`REVERTED` |The transaction passed validation but failed during execution in the sequencer. It is included in the block with the status `REVERTED`.
+Note: Since only INVOKE transactions have an execution phase, DEPLOY_ACCOUNT and DECLARE transactions cannot be reverted. They can be `REJECTED` if `+__VALIDATE_DEPLOY__+` or `+__VALIDATE_DECLARE__+`, respectively, fail when run in the sequencer. 
```

Comment:
@stoobie Can you please help to make this a note similar to the one in the rejected cell in the table?